docker/images/dev-docker/Dockerfile

```diff
@@ -0,0 +1,62 @@
+# Creates an image for skycoin development
+FROM skycoin/skycoindev-cli:develop
+
+# Install docker
+
+RUN set -ex; \
+    apt-get update; \
+    apt-get install -y --no-install-recommends \
+    btrfs-progs \
+    e2fsprogs \
+    iptables \
+    xfsprogs \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gnupg2 \
+    software-properties-common \
+    pigz ;\
+    apt-get clean
+
+RUN curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add -
+RUN apt-key fingerprint 0EBFCD88
+RUN add-apt-repository \
+   "deb [arch=amd64] https://download.docker.com/linux/debian \
+   $(lsb_release -cs) \
+   stable"
+
+RUN set -ex; \
+    apt-get update; \
+    apt-get install -y --no-install-recommends \
+    docker-ce ;\
+    apt-get clean
```

Comment:
After apt-get clean add `rm -rf /var/lib/apt/lists/*` It removes the packages cache, making the image smaller.